### ISSUE TITLE
SyntaxError: frappe-ui does not provide an export named 'frappeRequest'

### ISSUE THREAD

**Issue by osama1998H** [frappe/frappe-ui#58]
**Describe the bug**
When trying to use the frappeRequest export from the frappe-ui package in my Vue 3 project, Vite throws a SyntaxError stating that the requested module does not provide an export named 'frappeRequest'.

**To Reproduce**

1. Install frappe-ui package in a Vue 3 project.
2. Import frappeRequest from the frappe-ui package in main.js: `import { frappeRequest } from 'frappe-ui';`
3. Run the project using npm run dev or yarn dev.
4. See the following error in the console:
`SyntaxError: The requested module '/node_modules/.vite/frappe-ui.js?v=7a3651e8' does not provide an export named 'frappeRequest'`

**Expected behavior**
I expect the frappeRequest export to be available for use in my project without any errors.

**Environment**:

OS: [Linux]
Browser: [Chrome ]
Node.js version: [v16.19.1]
frappe-ui version: [0.0.89]
Vite version: [2.7.2]

**Additional context**
Please let me know if there is a workaround or if I am missing something in my setup.

**Comment by ishanloya** [comment]
Were you able to find a solution to this? @osama1998H  I have the same issue.

**Comment by NagariaHussain** [comment]
Hi! 

Are you guys on the latest version of frappe-ui? 

> frappe-ui version: [0.0.89]

This is an old version @osama1998H

**Comment by osama1998H** [comment]
@NagariaHussain yes i think i have the latest version and i tryed to update all npm packages but same issue

**Comment by NagariaHussain** [comment]
Okay, let me check and get back to you..
